bin/hdfs-mesos

```diff
@@ -4,4 +4,10 @@ if [ -z "$PORT0" ] ; then
   PORT0="8765"
 fi
 
-exec java -cp lib/*.jar -Dmesos.conf.path=etc/hadoop/mesos-site.xml -Dmesos.hdfs.config.server.port=$PORT0 org.apache.mesos.hdfs.Main
+if [ -z "$JAVA_HOME" ]; then
+  JAVA_CMD=java
```

Comment:
Should we maybe check `which java` to verify this is safe, otherwise error before trying to exec?